spring-cloud-consul-discovery/src/main/java/org/springframework/cloud/consul/discovery/HeartbeatProperties.java

```diff
@@ -51,6 +52,19 @@
 	@DecimalMax("0.9")
 	private double intervalRatio = 2.0 / 3.0;
 
+	/**
+	 * Whether or not to take the current system health (as reported via the Actuator
+	 * Health endpoint) into account when reporting the application status to the Consul
+	 * TTL check. Actuator Health endpoint also has to be available to the application.
+	 */
+	private boolean useActuatorHealth = true;
```

Comment:
If the property isn't use via the configuration properties object, add an entry in `META-INF/additional-spring-configuration-metadata.json`

Reply:
It is in the object [here](https://github.com/spring-cloud/spring-cloud-consul/pull/690/files/4201fab4694c4458186226a3355b00822a71d68d#diff-e6bb4a17dd56e87d1d96654ff47352cedaecbf3ad87577d1d53b08d988037880R107)

Reply:
I understand it's in the object there, but `isUseActuatorHealth()` is never called, so I don't see why it needs to be there and can't be additional metadata.

Reply:
Ah ok, got it. Yeh I will remove it and then move to the metadata file like the other "xyz.enabled" discovery props are.